src/server/qgsserverplugins.h

```diff
@@ -33,6 +33,10 @@ class QgsPythonUtils;
 class SERVER_EXPORT QgsServerPlugins
 {
   public:
+
+    /**
+     * Constructor.
```

Comment:
What do you think about introducing a test that avoids docstrings like this one ;)